WolvenKit.Modkit/RED4/Uncook.cs

```diff
@@ -785,26 +785,26 @@ public bool ExportMultiMeshWithRig(Dictionary<Stream, string> meshStreamS, List<
 
             var expMeshes = new List<RawMeshContainer>();
             var matData = new List<MatData>();
-            foreach (var meshStream in meshStreamS.Keys)
+            foreach (var meshStream in meshStreamS)
```

Comment:
what if this were like this?

```csharp
foreach (var (meshStream, meshName) in meshStreamS)
```

We could then leave the rest of the code untouched for all the `.Key` and instead of `.Value` it would be easier to understand that it's actually the name of the mesh.

Reply:
sounds good, I felt ugly when I was doing it. Was just trying to get it to work. Will make this change.

Reply:
> that would need to be (meshStream.Key, meshname) wouldnt it?

no. since `meshStreamS` (bad name btw lol) is a dictionary, doing a foreach loop does this:
```csharp
foreach (var (key, value) in dict)
```
so the first var becomes the key value which we we were previously iterating over